.travis.yml

```diff
@@ -89,3 +91,39 @@ script:
 
 after_failure:
   - php ./vendor/silverstripe/framework/tests/behat/travis-upload-artifacts.php --if-env BEHAT_TEST,ARTIFACTS_BUCKET,ARTIFACTS_KEY,ARTIFACTS_SECRET --target-path $TRAVIS_REPO_SLUG/$TRAVIS_BUILD_ID/$TRAVIS_JOB_ID --artifacts-base-url https://s3.amazonaws.com/$ARTIFACTS_BUCKET/ --artifacts-path ./artifacts/
+
+before_deploy:
+# Get asset-admin
+  - git clone https://github.com/silverstripe/silverstripe-asset-admin.git
+  - cd silverstripe-asset-admin
+  - yarn install
+  - cd ..
+  - mv silverstripe-asset-admin ../asset-admin
```

Comment:
You can wrap most of these commands in brackets for them to be executed in isolation from the current path, e.g. `(cd silverstripe-asset-admin && yarn install) && mv silverstripe-asset-admin ../asset-admin`